src/schema/me/lot_standing.js

```diff
@@ -72,10 +72,13 @@ export default {
   description: "The current user's status relating to bids on artworks",
   args: {
     artwork_id: {
-      type: new GraphQLNonNull(GraphQLString),
+      type: GraphQLString,
     },
     sale_id: {
-      type: new GraphQLNonNull(GraphQLString),
+      type: GraphQLString,
+    },
+    sale_artwork_id: {
```

Comment:
I think you need to update the loader call to pass it in?

Reply:
😓 updated.